src/VisualStudio/TestSetup/IntegrationTestServicePackage.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.Shell;
+
+namespace Roslyn.VisualStudio.Test.Setup
+{
+    [Guid(PackageGuidString)]
```

Comment:
Is this the only place `PackageGuidString` is used?  If so I'd rather it just be here, but if it's used elsewhere then this is fine.